hardware/products/ucm/uc5550.md

```diff
@@ -0,0 +1,257 @@
+# UC5550
+![G400D](images/g400d.jpg)
+# Overview
+---
+The UC5550 is a UCM that supports the STM F4 and F7 processor with 16MB of external RAM. It has a built-on STM WiFi module and an Ethernet PHY. It also has 4MB of external flash for large user programs.
+
+# Specifications
+---
+|					|							|
+| -----------------	| -------------------------	|
+| Processor			| STM 32-bit ARM Cortex-M4	|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+| Blah, blah, blah	| Blah, blah, blah			|
+
+# Resources
+---
+## Documents
+* [Schematic](http://files.ghielectronics.com/downloads/Schematics/Systems/G400D%20Dev%20Board%20Schematic.pdf)
+
+## Design Considerations
+This is where we will put the information that the engineer should be aware in order to make their board.  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+Blah, blah, blah  
+
+## Module Pin Configuration  
+| SO-DIMM Pin	| Universal Compute Standard	| Function Name									|
+| :-----------:	| :---------------------------:	| :-------------------------------------------:	|
+| 1				| AGND							| AGND											|
+| 2				| Ethernet TX-					| ETH PHY TX-									|
+| 3				| Bootstrap A					| BOOT											|
+| 4				| Ethernet TX+					| ETH PHY TX+									|
+| 5				| Analog VREF-					| Analog VREF-									|
+| 6				| Ethernet RX-					| ETH PHY RX-									| 
+| 7				| Reserved						| ---------------------------------------------	|
```

Comment:
N/A instead of ----

Reply:
That may make it way more cluttered looking. We leave a blank in the datasheet if there is nothing there. A blank on here may make it look incomplete.